src/kontrol/utils.py

```diff
@@ -112,3 +104,40 @@ def kontrol_file_contents() -> str:
 $ kontrol command --help
 ```
 """
+
+
+def kontrol_toml_file_contents() -> str:
+    return """[build.default]
+foundry-project-root       = '.'
+regen                      = true
+rekompile                  = true
+verbose                    = true
+debug                      = false
+require                    = 'lemmas.k'
+module-import              = 'TestBase:KONTROL-LEMMAS'
+
+[prove.default]
+foundry-project-root       = '.'
+verbose                    = true
+debug                      = false
+max-depth                  = 25000
+reinit                     = false
+cse                        = false
+workers                    = 4
+failure-information        = false
+counterexample-information = true
+minimize-proofs            = false
+fail-fast                  = true
+smt-timeout                = 1000
+break-every-step           = false
+break-on-jumpi             = false
+break-on-calls             = false
+break-on-storage           = false
+break-on-basic-blocks      = false
+break-on-cheatcodes        = false
+
```

Comment:
I understand it's not an an exhaustive list, but maybe we can add
```
run-constructor = false
```
too? Maybe that'll also serve as a reminder that they're not getting executed by default.

Reply:
Thanks! I'll add these!

Reply:
We also don't have `--max-iterations`, what does it default to internally?